src/colmap/controllers/incremental_pipeline.cc

```diff
@@ -502,25 +508,33 @@ void IncrementalPipeline::Reconstruct(
         ReconstructSubModel(mapper, mapper_options, reconstruction);
     switch (status) {
       case Status::INTERRUPTED: {
-        mapper.EndReconstruction(/*discard=*/false);
+        LOG(INFO) << "Keeping reconstruction due to interrupt";
+        mapper.EndReconstruction(/*discard=*/true);
```

Comment:
Is this change intended? Don't GUI users expect the reconstruction to not be discarded if they interrupt the process?

Reply:
Good catch. Unintended. The log message is already saying the right thing. :-)